conf/exports-wco-mapping.json

```diff
@@ -181,6 +181,10 @@
     "cds" : "locations.routingCountries.$1",
     "wco" : "42A.28A.62A.$1"
   },
+  {
+    "cds" : "consignment",
+    "wco" : "42A.28A"
+  },
```

Comment:
This is not needed. Just need to change the `wco` value above on line 182 to `"42A.28A"`